### ISSUE TITLE
Traffic Stats seems to exit when it can't talk to InfluxDB

### ISSUE THREAD

**Issue by smalenfant** [Comcast/traffic_control#1458]
Looks like Traffic Stats stopped here and gave up while I was doing maintenance on my InfluxDB cluster. 

```
[ERROR] 2016-05-25 19:43:40 Get http://cdn1cdidb0001:8086/ping?wait_for_leader=0s: dial tcp 10.22.250.31:8086: getsockopt: connection refused
[ERROR] 2016-05-25 19:43:40 Get http://cdn1cdidb0002:8086/ping?wait_for_leader=0s: dial tcp 10.22.250.32:8086: getsockopt: connection refused
```

**Comment by dneuman64** [comment]
I tried against the latest version and could not reproduce.  If Traffic Stats cannot connect to InfluxDb it will keep the data until InfluxDb is back up or it runs out of memory.  We should probably set a limit to how much data it holds onto, but it is possible that if your influxdb was down for an extended period of time that Traffic Stats OOMed.

I am going to close this issue, if you think it should be re-opened let me know.

**Comment by dneuman64** [comment]
Re-opening.  I could not reproduce when running locally (on mac), however I did see this issue with Traffic Stats running on a server.  Will look to fix for 1.8.0.
